spiralmatrix.py

```diff
@@ -1,53 +1,46 @@
-#time complexity: 0(n)
+#spiral matrix
+#time complexity : 0(n)
+# felt little difficult to track the row and columns
```

Comment:
thats okay it happens in the beginning with practice you will find all this very easy!!